worldedit-core/build.gradle.kts

```diff
@@ -27,6 +27,19 @@ dependencies {
     implementation("com.google.guava:guava")
     implementation("com.google.code.gson:gson")
 
+    implementation(libs.jchronic) {
+        exclude(group = "junit", module = "junit")
+    }
+    implementation("com.thoughtworks.paranamer:paranamer:2.6")
```

Comment:
Paranamer is built in to Java 8 onwards, we don't need to add it, hence we removed it years ago.

Reply:
This will be fixed in the next commit.